### PR TITLE
docs: day 03 and format

### PR DESCRIPTION
**Pull request by itsjoeoui**

### CODE REVIEW

File: docs/2024/puzzles/day03.md [hunk 1]
```diff
@@ -22,48 +22,57 @@ https://adventofcode.com/2024/day/3
...
+The shopkeeper at the North Pole Toboggan Rental Shop asks us to go through the input string, find all substrings that look like calls to a multiplication method `"mul(a,b)"`, perform the multiplications, and sum them together. To find such substrings, we will construct a regular expression that will look for the following:
+
 1. Each substring starts with a text "mul(". We need to escape the parenthesis in the regex form, so this will be `mul\(`.
 2. "mul(" is followed by the first positive integer. Since it's an integer and positive, we don't need to look for floating points or minuses; we can look for sequences of ciphers. In regex, one cipher is denoted by `\d`, and since we look for a sequence of one or more ciphers, it's `\d+`.
 3. After the first number there should be a comma, so `,`.
 4. Then there should be a second number, so again `\d+`.
 5. And the substring should finish with the closing parentheses, escaped again, so `\)`.
-That's it. Our whole regex looks like this: 
+   That's it. Our whole regex looks like this:
```
Comment (SethTisue) [inline docs/2024/puzzles/day03.md:32]: I think this was intended to be a new paragraph, not part of point 5?

File: docs/2024/puzzles/day03.md [hunk 2]
```diff
@@ -22,48 +22,57 @@ https://adventofcode.com/2024/day/3
...
 
 To implement this logic, first, we need a slightly more complicated regular expression that matches either "mul(a,b)" or "do()" or "don't()" and can extract from the input string all occurrences of each of those substrings in the order they appear.
+
 1. We already have the most complicated part of the new regular expressions - it's the regular expression from **Part 1**, `mul\((\d+)\,(\d+)\)`.
 2. Then, we need to be able to find a text "do()". As we know, we need to escape parentheses, so `do\(\)`.
 3. And similarly, we need to be able to find a substring "don't()" - `don't\(\)`.
 4. Finally, we must tell the compiler we are looking for substrings that match any of the three rules we made above. In regex, this is done by putting regular expressions in parentheses and separating them with a `|` sign, like this: `(GROUP 1|GROUP 2|GROUP 3)`.
-Together, our new regular expression looks as follows:
+   Together, our new regular expression looks as follows:
```
Comment (SethTisue) [inline docs/2024/puzzles/day03.md:62]: ditto, seems like a new paragraph?

File: docs/2024/puzzles/day03.md (hunk 1 above)
Comment (itsjoeoui) [reply docs/2024/puzzles/day03.md:32]: ah could be! fixing it now

File: docs/2024/puzzles/day03.md (hunk 2 above)
Comment (itsjoeoui) [reply docs/2024/puzzles/day03.md:62]: 👍

**Pull request merged**